### ISSUE TITLE
Screen reader: Improvement in the labels of some items present in the status bar.

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#96210]
Issue Type: <b>Feature Request</b>

When I request the contents of the status bar, the following is read by the orca.
'Open a remote window eita (Git) - master* No Problems Select a connection No Notifications Tweet Feedback Select Language Mode Select End of Line Sequence Select Encoding Select Indentation Go to Line/Column If you are not using a Screen Reader, please change the setting `editor.accessibilitySupport` to "off".'

In some cases what is being presented is the action to be taken and not the existing value. An example of this is the information of the row and column where the cursor is. Instead of reading the row and column number, the orca reads Go to Line / Column.

From @joanmarie
> I've not taken a look yet, but from your description it sounds like we want Isidor to add an aria-label value like "Ln 7, Col 16" on the element which has a title value of "Go to Line/Column."

CC @isidorn 

VS Code version: Code - Insiders 1.45.0-insider (a250df703de955a38aed427a917bce8278ab3331, 2020-04-24T09:57:20.729Z)
OS version: Linux x64 5.6.6-arch1-1

**Comment by isidorn** [comment]
@jvesouza thanks for filling this!
I have pushed a fix, so now it should be better. Please try it out from Tuesday and let us know how it behaves for you.

**Comment by jvesouza** [comment]
It really improved a lot. Perhaps it is a matter of preference, but it seems to me that each piece of information should end with a period. In my opinion this would facilitate the understanding of the information.
Something like the following:
Remote master*. 7 Errors. Select a connection. Notifications. Tweet Feedback. Prettier. JavaScript. LF. UTF-8. Spaces: 2. Ln 2, Col 1. Screen Reader Optimized.

Based on the example above, I was in doubt if the information 'Prettier JavaScript' is a single piece of information or if it is two pieces of information.

**Comment by joanmarie** [comment]
@jvesouza: I just committed a change to Orca to insert the pause breaks. If your punctuation level is not set to "all," Orca should pause.

Related aside: The reason why the punctuation level matters is that Orca currently accomplishes the pause by inserting a period. And hearing "dot" when all we wanted was a pause is annoying. Changing Orca so that pauses are not periods and thus will work no matter what the punctuation level is will be a slightly bigger change that I don't yet have time for. So for now, Orca doesn't insert pauses when the punctuation level is "all."

I personally think the place to address this need is in the screen readers rather than in the apps (like VSCode). We cannot count on every web app with a status bar to end the status bar items with a period. Thus the problem is bigger than VSCode.

Furthermore, I don't believe the other screen readers are taking advantage of the ARIA `code` role yet. As a result, I think those users will have to set their punctuation level to all in VSCode. Which means they might start hearing "dot" in between each status bar item. Which would kinda suck I think.

José: Please pull Orca master and see if the experience is better for you. Thanks!!

**Comment by jvesouza** [comment]
@joanmarie  It looks great! thank you.

**Comment by isidorn** [comment]
@joanmarie absolutely agree with you.
Screen readers should announce a pause between structured items (since these are different dom nodes it makes sense to have a puse in between). Adding a punctation on the vscode side does not make a lot of sense as you said as that might break other screen readers.

Thanks for fixing it that fast!

**Comment by isidorn** [comment]
Per José's comment adding verified label.
